src/Command/Project/ProjectCreateCommand.php

```diff
@@ -146,14 +169,37 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        $this->stdErr->writeln("The project is now ready!");
+        if ($template_options['initialize']) {
```

Comment:
Doesn't this need to be the $subscription->project_options at this point in the process?

Reply:
Not unless we're automatically initializing projects. This is looking for the `initialize` flag then from line 176 it's testing to make sure all the required parameters are being passed to successfully initialize the project.

Reply:
I've updated the process to automatically initialize any project that contains the required info to successfully initialize it.